tests/python/relax/test_remove_redundant_reshape.py

```diff
@@ -0,0 +1,94 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Test relax transform - Eliminate redundant reshape operations
+"""
+import tvm.testing
+from tvm import relax
+from tvm.relax.transform import DeadCodeElimination
+from tvm.relax.transform import RemoveRedundantReshape
+from tvm.script import ir as I, relax as R
+
+
+def _run_pass_compare_output(Before, Expected):
+    fused_mod = RemoveRedundantReshape()(Before)
+    assert relax.analysis.well_formed(fused_mod), "IRModule is not well-formed"
```

Comment:
Nitpick: I like the checks for well-formedness in general, but they are redundant here.  The test environment for relax unit tests already has a well-formed check before and after each transform.  ([location in `conftest.py`](https://github.com/apache/tvm/blob/unity/tests/python/relax/conftest.py#L23))

Reply:
Thanks for confirming that @Lunderberg! In that case we don't need well-formed checks, hence I have removed them.